Resources/doc/entity-listeners.rst

```diff
@@ -60,3 +57,34 @@ If you use a version of doctrine/orm < 2.5 you have to register the entity liste
 See also
 https://www.doctrine-project.org/projects/doctrine-orm/en/latest/reference/events.html#entity-listeners
 for more info on entity listeners and the resolver required by Symfony.
+
+
+Lazy Entity Listeners
+---------------------
+
+You can use the `lazy` attribute on the tag to make sure the listener services are only instantiated when they are actually used.
```

Comment:
You need to double quote keywords, and I think linkes should be wrapped to 72 chars.

```suggestion
You can use the ``lazy`` attribute on the tag to make sure the listener
services are only instantiated when they are actually used.
```